### PR TITLE
ci(engine): support verify ci for dataflow engine

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve?

Issue Number: ref #5352

### What is changed and how it works?

- Add verify ci entrypoint for dataflow engine in Makefile

#### Tests 

 - Unit test

##### Will it cause performance regression or break compatibility?

No

##### Do you need to update user documentation, design documentation or monitoring documentation?

No

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL, thanks! ❤️ @purelind

File: Makefile [hunk 1]
```diff
@@ -510,3 +510,14 @@ define run_engine_unit_test
 	|| { $(FAILPOINT_DISABLE); exit 1; }
 	$(FAILPOINT_DISABLE)
 endef
+
+engine_unit_test_in_verify_ci: check_failpoint_ctl tools/bin/gotestsum tools/bin/gocov tools/bin/gocov-xml
+	mkdir -p $(ENGINE_TEST_DIR)
+	$(FAILPOINT_ENABLE)
+	@export log_level=error; \
```
Comment (lance6716) [inline Makefile:517]: not sure if we need some info level log to debug when CI fails

Comment (amyangfei) [reply Makefile:517]: Seems this environment variable doesn't affect the log behavior in unit test.

**Review by maxshuang — APPROVED** [review]

**Pull request merged**